public/docs/_examples/structural-directives/dart/pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: structural_directives
+description: Structural directives example
```

Comment:
@zoechi I had actually but all other examples dont set it.
except this one
https://github.com/angular/angular.io/blob/master/public/docs/_examples/hierarchical-dependency-injection/dart/pubspec.yaml#L7